Utils/pom.xml

```diff
@@ -32,14 +32,15 @@
     <name>${project.artifactId}-${project.version}</name>
     <properties>
         <azuretool.version>3.58.0-SNAPSHOT</azuretool.version>
+        <azure.version>1.41.2</azure.version>
```

Comment:
why is this not needed before?

Reply:
I remove track1 sdk from maven repository